apps/api/src/app/change/usecases/promote-notification-template-change/promote-notification-template-change.usecase.ts

```diff
@@ -181,19 +181,7 @@ export class PromoteNotificationTemplateChange {
       return;
     }
 
-    await this.invalidateCache.invalidateByKey({
-      key: buildNotificationTemplateKey({
-        _id: newItem._id,
-        _environmentId: command.environmentId,
-      }),
-    });
-
-    await this.invalidateCache.invalidateByKey({
-      key: buildNotificationTemplateIdentifierKey({
-        templateIdentifier: newItem.triggers[0].identifier,
-        _environmentId: command.environmentId,
-      }),
-    });
```

Comment:
Previously, It was using the development environment Id and development environment's notification templates' id.
and was not invalidating for the Prod env